apisix/plugin.lua

```diff
@@ -264,7 +264,7 @@ function _M.load(config)
         local_conf, err = core.config.local_conf(true)
         if not local_conf then
             -- the error is unrecoverable, so we need to raise it
-            error("failed to load the configuration file: ", err)
+            error("failed to load the configuration file: " .. err)
```

Comment:
```suggestion
            core.log.error("failed to load the configuration file: ", err)
```

Reply:
We need to raise error here as it is unrecoverable. Otherwise, APISIX is still in trouble.

Reply:
got it